bundles/action/org.openhab.action.telegram/README.md

```diff
@@ -6,24 +6,23 @@ The Telegram Action service allows sending formatted messages to Telegram client
 
 As described in the Telegram Bot API, this is the manual procedure needed in order to get the necessary information.
 
-1. Create the Bot and get the Token
-  * On a Telegram client open a chat with BotFather.
-  * write `/newbot` to BotFather, fill all the needed information, write down the token. This is the authentication token needed.
-1. Create the destination chat and get the chatId
-  * Open a new chat with your new Bot and post a message on the chat
-  * Open a browser and invoke `https://api.telegram.org/bot<token>/getUpdates` (where `<token>` is the authentication token previously obtained)
-  * Look at the JSON result and write down the value of `result[0].message.chat.id`. That is the chatId.
+1.  Create the Bot and get the Token
+*   On a Telegram client open a chat with BotFather.
+*   write `/newbot` to BotFather, fill all the needed information, write down the token. This is the authentication token needed.
+2.  Create the destination chat and get the chatId
+*   Open a new chat with your new Bot and post a message on the chat
+*   Open a browser and invoke `https://api.telegram.org/bot<token>/getUpdates` (where `<token>` is the authentication token previously obtained)
+*   Look at the JSON result and write down the value of `result[0].message.chat.id`. That is the chatId.
 
 ## Configuration
 
 The action can be configured in `services/telegram.cfg`.
 
-| Property | Default | Required | Description |
-|----------|---------|:--------:|-------------|
-| bots | | Yes | Comma-separated list of `<bot-name>`s |
-| `<bot name>.chatId` | | Yes | chat id |
-| `<bot name>.token` | | Yes | authentication token |
-
+| Property            | Default | Required | Description                           |
+|---------------------|---------|-:-:------|---------------------------------------|

```

Comment:
This is broken.

Reply:
@9037568 I believe my latest commit fixes the table.  Please look, and if it is okay, please merge.

Reply:
Unfortunately, the latest commit does not make any changes to the table.